src/model/Model.cpp

```diff
@@ -6,93 +6,85 @@ Model::Model(string name) {
+    TextField * f = new TextField(fieldName, defaultValue, isPrimaryKey, isUnique, isNullable);
     fields.push_back(f);
     
     return f;
 }
 
-CharField* Model::charField(string name, int maxLength, string defaultValue, bool primary, bool unique, bool nullable)
-{
-    CharField * f = new CharField(name, maxLength, defaultValue, primary, unique, nullable);
+CharField* Model::charField(string fieldName, int maxLength, string defaultValue, bool isPrimaryKey, bool isUnique, bool isNullable) {
+    CharField * f = new CharField(fieldName, maxLength, defaultValue, isPrimaryKey, isUnique, isNullable);
     fields.push_back(f);
     
     return f;
 }
 
-BooleanField* Model::booleanField(string name, bool defaultValue, bool primary, bool unique, bool nullable)
-{
-    BooleanField * f = new BooleanField(name, defaultValue, primary, unique, nullable);
+BooleanField* Model::booleanField(string fieldName, bool defaultValue, bool isPrimaryKey, bool isUnique, bool isNullable) {
+    BooleanField * f = new BooleanField(fieldName, defaultValue, isPrimaryKey, isUnique, isNullable);
     fields.push_back(f);
     
     return f;
 }
 
-int Model::insert(Mode mode)
-{
+int Model::insert(Mode mode) {
     vector<Model*> models;
     models.push_back(this);
     
     return insertBatch(models, 1, mode);
 }
 
-int Model::insertBatch(vector<Model*> models, int batchsize, Mode mode)
-{
-    int size = models.size();
+int Model::insertBatch(vector<Model*> models, unsigned int batchsize, Mode mode) {
+    unsigned int size = models.size();
     
-    if(size < batchsize)
+    if (size < batchsize) {
         batchsize = size;
+    }
         
     //workout the size of the last batch
     int rem = size % batchsize;
-    int finalFullBatch = size - rem;    
+    unsigned int finalFullBatch = size - rem;    
     
     //generate SQL 
-    string sqlHead; // INSERT INTO table_name (fields) VALUES
-    string sqlBatch; // (?,?..,?)
-    string sqlUpdate; // ON DUPLICATE KEY UPDATE field=VALUES(field),...
+    string sqlHead; 
+    string sqlBatch; 
+    string sqlUpdate; 
     generateSQLParts(sqlHead, sqlBatch, sqlUpdate, mode);
     
     string sqlTail = generateSQLTail(sqlBatch, sqlUpdate, batchsize, mode);
     
     //cout << sqlHead + sqlTail << endl;
     
     //prepare first statement
-    conn.prepareStatement(sqlHead + sqlTail);
+    _connection.prepareStatement(sqlHead + sqlTail);
     
     int paramNum = 0;
     bool finalBatch = false;
     //iterate through the list of models inserting batches of the specified size
-    for (int i = 0; i < models.size(); i++)
-    {    
+    for (unsigned int i = 0; i < models.size(); i++) {    
         Model * m = models[i];
         //iterate through the list of fields, adding them as parameters
-        for (int j = 0; j < m->fields.size(); j++) {
+        for (unsigned int j = 0; j < m->fields.size(); j++) {
             Field * f = m->fields[j];
             
             bool ignoreField = f->isAutoFilled();
```

Comment:
You can actually just remove this line and use `f->isAutoFilled()` in the if-statement.